docs/domains/index.html

```diff
@@ -1158,6 +1158,10 @@
           "description": "If present, every time this value is updated the operator will restart the required servers.",
           "type": "string"
         },
+        "dataHome": {
+          "description": "The in-pod name of the directory for data storage of default and custom file stores.",
```

Comment:
Maybe we can provide a more detail here? For example, change "The" to "Optional", and add a sentence something like: "If dataHome is not set or is set to an empty string, data storage directories are determined from WebLogic domain home configuration.".

Reply:
Confirmed resolved.